MapboxCoreNavigation/CLHeading.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import CoreLocation
+
+extension CLHeading {
+    var preferredHeading: CLLocationDirection {
+        return magneticHeading
```

Comment:
Ideally `trueHeading` over `magneticHeading`, espeically if the `headingAccuracy` is high